tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -57,18 +57,39 @@ def get_config_help(self):
         }
         return config_help
 
-    def set_destinations(self):
```

Comment:
Is there any specific reason why this overriding method is removed and the same functionality is moved to get_user_ids/populate_alert_destinations? The whole purpose NotificationPlugin is ro provide a generic interface.

Reply:
actually get user id function returning only list of user id, but it is not a destination, the destination are list of email ids. so i renamed this function. like get_user_id